app/services/album.js

```diff
@@ -30,3 +31,31 @@ exports.getPhotosOfAlbum = async albumId => {
     throw errors.conectionError(e.message);
   }
 };
+
+exports.buyAlbum = async (albumId, userEmail) => {
+  const endpoint = `${config.api_url}/albums/${albumId}`;
+  try {
+    const user = await db.users.find({ where: { email: userEmail } });
+    const response = await fetch(endpoint);
```

Comment:
the connection with other API is responsability of controller or another service that is called from a controllar